docs/general/endpoints.rst

```diff
@@ -0,0 +1,163 @@
+Endpoints
+---------
+
+When a model is deployed from a config file, it results in a ML
+server capable of the following paths:
+
+All paths listed below should be prefixed with ``/gordo/v0/<project-name>/<model-name>``
```

Comment:
Maybe a slightly different wording. From the model-servers perspective this is not the required path right, it could actually be served anywhere. But maybe a sentence that in a "normal" deployment or something like that?